### PR TITLE
fix typos in lights_and_shadows.rst

### PR DESCRIPTION
**Pull request by flintsteel7**

### CODE REVIEW

File: tutorials/3d/lights_and_shadows.rst [hunk 1]
```diff
@@ -109,7 +109,7 @@ generally work well, but tweaking the first split a bit is common to give more
 detail to close objects (like a character in a third person game).
 
 Always make sure to set a shadow *Max Distance* according to what the scene needs.
-The closer the max distance, the higher quality they shadows will have.
+The closer the max distance, the higher quality the shadows will have.
```
Comment (Calinou) [inline tutorials/3d/lights_and_shadows.rst]: I'd suggest rewording the sentence entirely:

```suggestion
A lower maximum distance will result in better-looking shadows.
```

Comment (flintsteel7) [reply tutorials/3d/lights_and_shadows.rst]: Absolutely, thanks!

**Comment by akien-mga** [comment]
Thanks!
